### PR TITLE
Abc feature

### PR DESCRIPTION
**Pull request by cjannes-tibco**
pls review that boss

### CODE REVIEW

File: abc/a [hunk 1]
```diff
@@ -0,0 +1 @@
+yo
```
Comment (clemja) [inline abc/a:1]: that's makes no sense... I love it

File: abc/b [hunk 2]
```diff
@@ -0,0 +1 @@
+lo
```
Comment (clemja) [inline abc/b:1]: good one

File: abc/c [hunk 3]
```diff
@@ -0,0 +1 @@
+ahah
```
Comment (clemja) [inline abc/c:1]: haha that's funny yeah

Comment (clemja) [reply abc/c:1]: Could you please replace ahah by haha ? it's better I think

**Review by clemja — CHANGES_REQUESTED** [review]
Please my boy see my comments on file c and could you please add d file with something great?

other files are master piece thank's

File: abc/a (hunk 1 above)
Comment (cjannes-tibco) [reply abc/a:1]: Thank you boss !

File: abc/c (hunk 3 above)
Comment (cjannes-tibco) [reply abc/c:1]: I will commit it right away ! thank you boss !

**Review by cjannes-tibco — COMMENTED** [review]
Ok

**Pull request merged**